pkg/otelcollector/consts/consts.go

```diff
@@ -87,10 +87,14 @@ const (
 	ApertureFlowStatusOK = "OK"
 	// ApertureFlowStatusError const for error status.
 	ApertureFlowStatusError = "Error"
-	// ApertureCacheLookupStatusLabel describes status of the cache lookup.
-	ApertureCacheLookupStatusLabel = "aperture.cache_lookup_status"
-	// ApertureCacheOperationStatusLabel describes status of the cache operation.
-	ApertureCacheOperationStatusLabel = "aperture.cache_operation_status"
+	// ApertureResultCacheLookupStatusLabel describes status of the cache lookup.
+	ApertureResultCacheLookupStatusLabel = "aperture.cache_lookup_status"
+	// ApertureResultCacheOperationStatusLabel describes status of the cache operation.
+	ApertureResultCacheOperationStatusLabel = "aperture.cache_operation_status"
+	// ApertureGlobalCacheLookupStatusLabel describes status of the cache lookup.
+	ApertureGlobalCacheLookupStatusLabel = "aperture.global_cache_lookup_status"
+	// ApertureGlobalCacheOperationStatusLabel describes status of the cache operation.
+	ApertureGlobalCacheOperationStatusLabel = "aperture.global_cache_operation_status"
```

Comment:
@coderabbitai have these metrics been documented properly?

Reply:
Did you search all markdown files?

Reply:
Why are you searching for the variable names? We have to search the labels.

Reply:
Can you guide me where the documentation changes have to be made for the metrics?

Reply:
Can you add the documentation in that file and provide me the diff?

Reply:
@coderabbit This is not correct place. These are flow metrics, not prometheus metrics.